src/router.ts

```diff
@@ -57,6 +57,7 @@ if (domain) {
   routes.push(
     { path: '/', name: 'home', component: Space, children: spaceRoutes },
     { path: '/delegate/:key?/:to?', name: 'delegate', component: Delegate },
+    { path: '/playground/:name', name: 'playground', component: Playground },
```

Comment:
Ahhh so I actually shouldn't have removed this.... Wasn't aware that there actually IS a way to access that playground from a custom domain.

![image](https://user-images.githubusercontent.com/6792578/151966129-086ffa23-d349-4398-8f0a-0a7317dbb778.png)

Sorry for that! :D

Reply:
Discussed in voice :)